internal/cli/profile.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2024 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"github.com/mongodb/mongodb-atlas-cli/internal/config"
+	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
+)
+
+func InitProfile(profile string) error {
```

Comment:
[q] should we test this method? is it even testable?

Reply:
I don't think this method is testable since it's calling "static" methods in the `config` module.
Unless there's a way to mock those easily in go?

Reply:
yeah that's why I asked the "testable" to make it testable we'd need to to a bit of refactor on the signature to accept the config object

Reply:
I could do that, but then I would have to add `List()` to the `Profile` struct so it can be mocked. Would that be ok?

Reply:
I'll say yes but the more I think about it I wonder how hacky it may be, feel free to discard this thread or take it as a future improvement if you want to play a little